tests/conftest.py

```diff
@@ -12,11 +15,21 @@
 # The profile dictionary, used to write out profiles.yml
 @pytest.fixture(scope="class")
 def dbt_profile_target():
-    return {
+    profile = {
         'type': 'flink',
         'threads': 1,
         'host': os.getenv('FLINK_SQL_GATEWAY_HOST', '127.0.0.1'),
         'port': int(os.getenv('FLINK_SQL_GATEWAY_PORT', '8083')),
         'session_name': os.getenv('SESSION_NAME', 'test_session'),
-        'database': os.getenv('DATABASE_NAME', 'test_db'),
+        'database': os.getenv('DATABASE_NAME', 'default_catalog'),
+        'schema': os.getenv('DATABASE_NAME', 'default_database'),
     }
+
+    # we need setup MockFlinkSqlGatewayClient
+    MockFlinkSqlGatewayClient.create_session(
```

Comment:
Can we make it to create and use instant instead of using it like singleton? 
This will lead to problems in case of running tests in parallel or just not cleaning after in some single test. 
Tests by nature should be isolated from each other as much as possible.

Reply:
you are absolutely right, i'll handle it

Reply:
done